run.py

```diff
@@ -577,18 +581,23 @@ def run(args):
         if test.get('recreate-cluster') is True:
             ceph_cluster_dict, clients = create_nodes(conf, inventory, osp_cred, run_id, service, instances_name)
         tcs.append(tc)
+    url_base = "http://magna002.ceph.redhat.com/cephci-jenkins"
+    run_dir_name = run_dir.split('/')[-1]
+    log.info("\nAll test logs located here: {base}/{dir}".format(base=url_base, dir=run_dir_name))
     close_and_remove_filehandlers()
     if post_to_report_portal:
         service.finish_launch(end_time=timestamp())
         service.terminate()
     if xunit_results:
         create_xunit_results(suite_name, tcs, run_dir)
-    url_base = "http://magna002.ceph.redhat.com/cephci-jenkins"
-    run_dir_name = run_dir.split('/')[-1]
     print("\nAll test logs located here: {base}/{dir}".format(base=url_base, dir=run_dir_name))
     print_results(tcs)
     send_to_cephci = post_results or post_to_report_portal
-    email_results(tcs, run_id, send_to_cephci)
+    run_end_time = datetime.datetime.now()
+    total_run_time = (datetime.datetime.now() - run_start_time)
+    total_time = """'''Start Time:''' {a} '''End Time:''' {b} '''Total duration:'''{c} \
```

Comment:
I would suggest individual row for Start/End/Total time/duration.

Reply:
i believe the general information table is already big enuf with 11 rows( gen info on par with actual test suite TC rows)